src/main/java/uk/co/onsdigital/discovery/metadata/api/dto/ResultPage.java

```diff
@@ -44,7 +44,11 @@ public int getPage() {
     }
 
     public int getTotalPages() {
-        return (int) (getTotal() / getItemsPerPage());
+        int pages = (int) (getTotal() / getItemsPerPage());
+        if (getTotal() % getItemsPerPage() > 0) {
+            pages++;
+        }
+        return pages;
```

Comment:
You can also do this as: `return (int) (getTotal() +  getItemsPerPage() - 1) / getItemsPerPage();`